wine-staging/PKGBUILD

```diff
@@ -0,0 +1,172 @@
+  freetype2       #lib32-freetype2
+  gcc-libs        #lib32-gcc-libs
+  gettext         #lib32-gettext
+  libpcap         #lib32-libpcap
+  libxcursor      #lib32-libxcursor
+  libxi           #lib32-libxi
+  libxrandr       #lib32-libxrandr
+)
+makedepends=(autoconf bison perl flex mingw-w64-gcc
+  git
+  alsa-lib              #lib32-alsa-lib
+  ffmpeg
+  giflib                #lib32-giflib
+  gnutls                #lib32-gnutls
+  gst-plugins-base-libs #lib32-gst-plugins-base-libs
+  gtk3                  #lib32-gtk3
+  libcups               #lib32-libcups
+  libgphoto2
+  libpulse              #lib32-libpulse
+  libva                 #lib32-libva
+  libxcomposite         #lib32-libxcomposite
+  libxinerama           #lib32-libxinerama
+  libxxf86vm            #lib32-libxxf86vm
+  mesa                  #lib32-mesa
+  mesa-libgl            #lib32-mesa-libgl
+  opencl-headers
+  opencl-icd-loader     #lib32-opencl-icd-loader
+  python
+  samba
+  sane
+  sdl2                  #lib32-sdl2
+  v4l-utils             #lib32-v4l-utils
+  vulkan-icd-loader     #lib32-vulkan-icd-loader
+)
+makedepends+=(
+  ntsync-header
+)
+optdepends=(
+  alsa-lib              #lib32-alsa-lib
+  alsa-plugins          #lib32-alsa-plugins
+  cups                  #lib32-libcups
+  dosbox
+  ffmpeg
+  giflib                #lib32-giflib
+  gnutls                #lib32-gnutls
+  gst-plugins-base-libs #lib32-gst-plugins-base-libs
+  gtk3                  #lib32-gtk3
+  libgphoto2
+  libpulse              #lib32-libpulse
+  libva                 #lib32-libva
+  libxcomposite         #lib32-libxcomposite
+  libxinerama           #lib32-libxinerama
+  opencl-icd-loader     #lib32-opencl-icd-loader
+  samba
+  sane
+  sdl2                  #lib32-sdl2
+  v4l-utils             #lib32-v4l-utils
+  vulkan-icd-loader     #lib32-vulkan-icd-loader
+  wine-gecko
+  wine-mono
+)
+optdepends+=(
+  NTSYNC-MODULE
+)
+
+provides=("wine=$pkgver")
+conflicts=('wine')
+install=wine.install
+
+prepare() {
+  # Get rid of old build dirs
+  rm -rf $pkgname-{32,64}-build
+  mkdir $pkgname-{32,64}-build
+
+  cd wine
+  # apply wine-staging patchset
+  ../wine-staging/staging/patchinstall.py --backend=git-apply --all \
+    --no-autoconf \
+    -W eventfd_synchronization \
+    -W server-PeekMessage \
+    -W server-Signal_Thread
+
+  patch -Np1 -i "$srcdir"/wine-ntsync.patch
+
+  ./tools/make_requests
+  ./dlls/winevulkan/make_vulkan
+  ./tools/make_specfiles
+  autoreconf -fiv
+}
+
+build() {
+  # Doesn't compile without remove these flags as of 4.10
+  # incompatible-pointer-types: https://bugs.gentoo.org/919758
+  export CFLAGS="$CFLAGS -ffat-lto-objects -Wno-error=incompatible-pointer-types"
+
+  # Apply flags for cross-compilation
+  export CROSSCFLAGS="-O2 -pipe"
+  export CROSSCXXFLAGS="-O2 -pipe"
+  export CROSSLDFLAGS="-Wl,-O1"
+
+  echo "Building Wine-64..."
+  cd "$srcdir/$pkgname-64-build"
+  ../wine/configure \
+    --prefix=/usr \
+    --libdir=/usr/lib \
+    --with-x \
```

Comment:
I'm not sure if ffmpeg is actually used by default, so perhaps we should add ``--with-ffmpeg``here?

Reply:
Good point, so I checked, according to `configure` it should at least checked and found
```
checking for libavutil/avutil.h... yes
checking for libavformat/avformat.h... yes
checking for libavcodec/avcodec.h... yes
checking for libavcodec/bsf.h... yes
checking for av_log_set_callback in -lavutil... yes
checking for av_find_input_format in -lavformat... yes
checking for avcodec_get_name in -lavcodec... yes
```

Reply:
Then it's okay, I guess.